specs/commandline/list.md

```diff
@@ -29,6 +29,7 @@ Aliases:
 Flags:
   -d, --debug             debug mode
   -h, --help              help for list
+      --local-artifact    [Preview] if set, list signatures associated with the image in OCI layout directory
```

Comment:
I saw the flag naming is different from the [doc](https://hackmd.io/@patrickzheng200/B1F82vi6o) and [PR](https://github.com/notaryproject/notation/pull/595), is it intended?

Reply:
> I found the flag naming is different from the [doc](https://hackmd.io/@patrickzheng200/B1F82vi6o) and [PR](https://github.com/notaryproject/notation/pull/595), is it intended?

Yeah, I think `--local-artifact` is better. I'll change to `--local-artifact` in the PR.